### ISSUE TITLE
svg tags not showing when using node! macro in the client

### ISSUE THREAD

**Issue by ivanceras** [ivanceras/sauron#34]
Svg elements is not showing when used in browser. This would be rendered correctly when used server-side.

```html
  <svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
    <defs>
      <pattern id="Pattern" x="0" y="0" width=".25" height=".25">
        <rect x="0" y="0" width="50" height="50" fill="skyblue"/>
      </pattern>
    </defs>

    <rect fill="url(#Pattern)" stroke="black" width="200" height="200"/>
  </svg>

```

**Comment by ivanceras** [comment]
This is because svg elements need to be created using `document.create_element_ns`. This is mitigated by exposing the `sauron-parse` feature which allows us to determine the namespace of a tag via lookup.

**Comment by ivanceras** [comment]
Fixed in commit 9411a82bffaf4891c8acfa84d24f635360761d66
